Tools/WinMLRunner/src/CommandLineArgs.cpp

```diff
@@ -213,7 +218,21 @@ CommandLineArgs::CommandLineArgs(const std::vector<std::wstring>& args)
             PrintUsage();
             return;
         }
-    }
+        else if ((_wcsicmp(args[i].c_str(), L"-ConcurrentLoad") == 0))
+        {
+            ToggleConcurrentLoad(true);
+        }
+        else if ((_wcsicmp(args[i].c_str(), L"-NumThreads") == 0))
+        {
+            unsigned num_threads = std::stoi(args[++i].c_str());
+            SetNumThreads(num_threads);
+        }
+        else if ((_wcsicmp(args[i].c_str(), L"-ThreadInterval") == 0))
+        {
+            unsigned thread_interval = std::stoi(args[++i].c_str());
```

Comment:
Same thing here, I think we should add an extra check to check that the next argument is not another flag but an integer.